pkg/cdi/registry.go

```diff
@@ -122,15 +122,12 @@ var (
 // GetRegistry returns the CDI registry. If any options are given, those
 // are applied to the registry.
 func GetRegistry(options ...Option) Registry {
-	var new bool
 	initOnce.Do(func() {
-		reg = &registry{newCache(options...)}
-		new = true
+		reg = &registry{GetDefaultCache()}
```

Comment:
Should we document somewhere that the registry actually uses the default cache as the backend (manipulating either of those changes the other, too)?

Reply:
The documentation updates in this PR state that the registry operates on the default cache.